tracing-journald/src/lib.rs

```diff
@@ -188,14 +188,20 @@ struct SpanVisitor<'a> {
     prefix: Option<&'a str>,
 }
 
+#[derive(Copy, Clone)]
+enum FieldValue<'a> {
+    Raw(&'a str),
+    Debug(&'a dyn fmt::Debug),
+}
+
 impl Visit for SpanVisitor<'_> {
```

Comment:
Should we specialize `record_str` here too?

Reply:
Oh, sure, I forgot this part.  Thanks for reminding me :pray: